lib/Runtime/Debug/DiagObjectModel.h

```diff
@@ -353,10 +354,20 @@ namespace Js
                             }
                             else
                             {
+                                PropertyIndex propertyIndex = activeScopeObject->GetPropertyIndex(resolveObject.propId);
```

Comment:
This will always return invalid property index as activeScopeObject doens't have this propId. We need to call GetPropertyIndex on resolveObject.obj or resolveObject.originalInstance